R/descriptives.R

```diff
@@ -289,10 +288,10 @@ Descriptives <- function(jaspResults, dataset, options) {
   stats$addColumnInfo(name="Valid",   title=gettext("Valid"), type="integer")
   stats$addColumnInfo(name="Missing", title=gettext("Missing"), type="integer")
 
+  if (options$mode)                 stats$addColumnInfo(name="Mode",                        title=gettext("Mode"),                    type="number")
+  if (options$median)               stats$addColumnInfo(name="Median",                      title=gettext("Median"),                  type="number")
   if (options$mean)                 stats$addColumnInfo(name="Mean",                        title=gettext("Mean"), 				            type="number")
```

Comment:
The order here is the only thing that matters, so I think these two lines (and the `if (options$sum) ...`) are the only required changes.